bindings/pyroot/src/Pythonize.cxx

```diff
@@ -2504,7 +2505,7 @@ Bool_t PyROOT::Pythonize( PyObject* pyclass, const std::string& name )
 
    }
 
-   else if ( IsTemplatedSTLClass( name, "vector" ) ) {
+   else if ( IsTemplatedSTLClass( name, "vector" ) || (name.find("ROOT::VecOps::RVec<") != std::string::npos) ) {
```

Comment:
Shouldn't this be "starting with "ROOT::VecOps::RVec<" ?

Reply:
I mean, what if "MyOddTemplate<ROOT::VecOps::RVec>"?

Reply:
Fixed.